.evergreen/scripts/compile-std.sh

```diff
@@ -137,6 +137,10 @@ echo "configure_flags: ${configure_flags[*]}"
 # Use ccache if able.
 . "${script_dir:?}/find-ccache.sh"
 find_ccache_and_export_vars "$(pwd)" || true
+if [[ "${OSTYPE:?}" == cygwin ]]; then
```

Comment:
Suggest invoking `find_ccache_and_export_vars` (I expect ccache is not applied in this file).

Reply:
Call to `find_ccache_and_export_vars` is done above on line 136.

Reply:
Done.